vulnerabilities/importers/apache_kafka.py

```diff
@@ -9,118 +9,321 @@
+        "action": "include",
+        "2.8.0 - 2.8.1, 3.0.0 - 3.0.1, 3.1.0 - 3.1.1, 3.2.0 - 3.2.1": "affected",
+        "2.8.2, 3.0.2, 3.1.2, 3.2.3": "fixed",
+        "affected_version_range": "vers:maven/>=2.8.0|<=2.8.1|!=2.8.2|>=3.0.0|<=3.0.1|!=3.0.2|>=3.1.0|<=3.1.1|!=3.1.2|>=3.2.0|<=3.2.1|!=3.2.3",
+    },
+    "CVE-2022-23302": {
+        "action": "omit",
+    },
+    "CVE-2022-23305": {
+        "action": "omit",
+    },
+    "CVE-2022-23307": {
+        "action": "omit",
+    },
+    "CVE-2021-45046": {
+        "action": "omit",
+    },
+    "CVE-2021-44228": {
+        "action": "omit",
+    },
+    "CVE-2021-4104": {
+        "action": "omit",
+    },
+    "CVE-2021-38153": {
+        "action": "include",
+        "2.0.0, 2.0.1, 2.1.0, 2.1.1, 2.2.0, 2.2.1, 2.2.2, 2.3.0, 2.3.1, 2.4.0, 2.4.1, 2.5.0, 2.5.1, 2.6.0, 2.6.1, 2.6.2, 2.7.0, 2.7.1, 2.8.0.": "affected",
+        "2.6.3, 2.7.2, 2.8.1, 3.0.0 and later": "fixed",
+        "affected_version_range": "vers:maven/2.0.0|2.0.1|2.1.0|2.1.1|2.2.0|2.2.1|2.2.2|2.3.0|2.3.1|2.4.0|2.4.1|2.5.0|2.5.1|2.6.0|2.6.1|2.6.2|!=2.6.3|2.7.0|2.7.1|!=2.7.2|2.8.0.|!=2.8.1|<3.0.0",
+    },
+    "CVE-2019-12399": {
+        "action": "include",
+        "2.0.0, 2.0.1, 2.1.0, 2.1.1, 2.2.0, 2.2.1, 2.3.0": "affected",
+        "2.2.2, 2.3.1 and later": "fixed",
+        "affected_version_range": "vers:maven/2.0.0|2.0.1|2.1.0|2.1.1|2.2.0|2.2.1|!=2.2.2|2.3.0|<2.3.1",
+    },
+    "CVE-2018-17196": {
+        "action": "include",
+        "0.11.0.0 to 2.1.0": "affected",
+        "2.1.1 and later": "fixed",
+        "affected_version_range": "vers:maven/>=0.11.0.0|<=2.1.0|<2.1.1",
+    },
+    "CVE-2018-1288": {
+        "action": "include",
+        "0.9.0.0 to 0.9.0.1, 0.10.0.0 to 0.10.2.1, 0.11.0.0 to 0.11.0.2, 1.0.0": "affected",
+        "0.10.2.2, 0.11.0.3, 1.0.1, 1.1.0": "fixed",
+        "affected_version_range": "vers:maven/>=0.9.0.0|<=0.9.0.1|>=0.10.0.0|<=0.10.2.1|!=0.10.2.2|>=0.11.0.0|<=0.11.0.2|!=0.11.0.3|1.0.0|!=1.0.1|!=1.1.0",
+    },
+    "CVE-2017-12610": {
+        "action": "include",
+        "0.10.0.0 to 0.10.2.1, 0.11.0.0 to 0.11.0.1": "affected",
+        "0.10.2.2, 0.11.0.2, 1.0.0": "fixed",
+        "affected_version_range": "vers:maven/>=0.10.0.0|<=0.10.2.1|!=0.10.2.2|>=0.11.0.0|<=0.11.0.1|!=0.11.0.2|!=1.0.0",
+    },
+}
 
 
 class ApacheKafkaImporter(Importer):
+
+    GH_PAGE_URL = "https://raw.githubusercontent.com/apache/kafka-site/asf-site/cve-list.html"
+    ASF_PAGE_URL = "https://kafka.apache.org/cve-list"
+    spdx_license_expression = "Apache-2.0"
+    license_url = "https://www.apache.org/licenses/"
+
     @staticmethod
-    def fetch_advisory_page():
-        page = requests.get(GH_PAGE_URL)
+    def fetch_advisory_page(self):
+        page = requests.get(self.GH_PAGE_URL)
         return page.content
 
-    def set_api(self):
-        self.version_api = GitHubTagsAPI()
-        asyncio.run(self.version_api.load_api(["apache/kafka"]))
+    # For now, don't use the GH API
```

Comment:
Do not leave unused code around in comments here and below

Reply:
Done.